src/vellum/workflows/nodes/core/try_node/node.py

```diff
@@ -56,34 +56,60 @@ class TryNode(BaseNode[StateType], Generic[StateType], metaclass=_TryNodeMeta):
     class Outputs(BaseNode.Outputs):
         error: Optional[VellumError] = None
 
-    def run(self) -> Outputs:
+    def run(self) -> Iterator[BaseOutput]:
```

Comment:
rewrote this method to work more like subworkflow nodes, since we need to handle streaming